ChangeLog.old

```diff
@@ -249,7 +249,7 @@
 
 	* Released version 0.31
 
-	* Makefile.am (install-root): Neede to make nullmailer-queue
+	* Makefile.am (install-root): Needed to make nullmailer-queue
```

Comment:
I can't tell if this should be `need` or `needed`

Reply:
Most likely needed -- the entries in the ChangeLog (derived from svn) were typically past tense (as opposed to the git log, which is present tense).